README.md

```diff
@@ -1 +1,176 @@
-# Ruby-capstone-proj
\ No newline at end of file
+<div align="center">
+  <h3><b>Ruby Capstone Project: Catalog of my Things README</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 CATALOG_OF_THINGS <a name="about-project"></a>
+
+**Catalog of things ** is a console application that will help you to keep a record of different types of things you own: books, music albums, movies, and games.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://ruby-lang.org/">Ruby</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Add a new game, a music album and a book
+- Add extra details such genre, label and authors
+- All the data is persisted in JSON files
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- A text editor preferably [Visual Studio code](https://code.visualstudio.com/)
+- Latest version of [Ruby](https://www.ruby-lang.org/en/downloads/) installed
+- [PostgreSQL Server](https://www.postgresql.org/download/)
+
+
+### Setup
+
+Clone this [repository](https://github.com/Ngufor-emmanuella/Ruby-capstone-proj) to your desired folder:
+
+```sh
+  
+  git clone https://github.com/Ngufor-emmanuella/Ruby-capstone-proj.git
+  cd Ruby-capstone-proj
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command in your terminal:
+
+```sh
+  ruby src/main.rb
```

Comment:
- [ ] Nice work done on your Readme, kindly change this `ruby main.rb` it throws an error when I used `ruby src/main.rb`.

Reply:
Hi @danielufeli we have updated the command as you suggested.